usl_models/requirements.txt

```diff
@@ -8,7 +8,7 @@ gast==0.5.4
 google-api-core==2.19.1
 google-auth==2.32.0
 google-auth-oauthlib==1.0.0
-google-cloud-aiplatform==1.43.0
+google-cloud-aiplatform[prediction]==1.43.0
```

Comment:
What additional packages does the `[prediction`] bit pull in? They should make it into this requirements.txt as well. We'll also need to update the setup.py, as that's what's used when we deploy this code into training land, not the requirements.txt (we should maybe remove the requirements.txt as it's not used to deploy anything at the moment) to make clear the `setup.py` is the important bit.

Reply:
This is required for prediction_util which lets us download the model artifacts from artifact registry. I'll move this to setup.py.

Reply:
Ack, removed